web/app/scripts/recent-proportions/recent-proportions-controller.js

```diff
@@ -0,0 +1,19 @@
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function RecentProportionsController() {
+        var ctl = this;
+
+        ctl.proportions = [
+          { proportion: 'About half', label: 'birds' },
+          { proportion: 'Some', label: 'frozen waffles' },
+          { proportion: '1.2%', label: 'covers of \'Freebird\'' }
```

Comment:
Cute placeholders, but I think it'd be better to use the old placeholders for demo purposes.